packages/svelte/src/compiler/phases/3-transform/css/index.js

```diff
@@ -333,7 +333,7 @@ function is_empty(rule) {
 		}
 
 		if (child.type === 'Atrule') {
-			return false; // TODO
+			if (child.block === null || child.block.children.length > 0) return false;
```

Comment:
Should this be `child.block !== null`? How does a media query with a null block looks like?

Reply:
and at rule can also be something like `@import ...` which means _something_ is happening within the rule